operations/host_spawn.go

```diff
@@ -553,17 +555,28 @@ func hostModifyVolume() cli.Command {
 				Name:  joinFlagNames(sizeFlag, "s"),
 				Usage: "set new volume `SIZE` in GiB",
 			},
+			cli.DurationFlag{
+				Name:  extendFlag,
+				Usage: "extend volume's expiration",
+			},
+			cli.BoolFlag{
+				Name:  noExpireFlagName,
+				Usage: "make volume never expire",
+			},
 		},
 		Before: mergeBeforeFuncs(
 			setPlainLogger,
 			requireStringFlag(idFlagName),
-			requireAtLeastOneFlag(displayNameFlagName, sizeFlag),
+			requireAtLeastOneFlag(displayNameFlagName, sizeFlag, extendFlag, noExpireFlagName),
+			mutuallyExclusiveArgs(false, extendFlag, noExpireFlagName),
```

Comment:
Can you add mutuallyExclusiveArgs to to host modify?

Reply:
done